src/scrapers/visa-cal.ts

```diff
@@ -502,6 +519,7 @@ class VisaCalScraper extends BaseScraperWithBrowser<ScraperSpecificCredentials>
 
         return {
           txns,
+          balance: -frame?.nextTotalDebit,
```

Comment:
If `frame` is `underfined`, the `balance` will be `NaN`. Please be more specific like:


```suggestion
          balance: _.isNumber(frame?.nextTotalDebit) ? -frame?.nextTotalDebit : undefined,
```